dist/PublicLab.Editor.js

```diff
@@ -21960,11 +21960,19 @@ module.exports = function initTables(_module, wysiwyg) {
   });
 
   var builder  = '<div class="form-inline form-group ple-table-popover" style="width:400px;">';
-      builder += '<input value="4" class="form-control rows" style="width:75px;" />';
-      builder += ' x ';
-      builder += '<input value="3" class="form-control cols" style="width:85px;" /> ';
-      builder += '<a class="ple-table-size btn btn-default"><i class="fa fa-plus"></i></a>';
-      builder += '</div>';
+        builder += '<a id="decRows" class="btn btn-sm btn-default"><i class="fa fa-minus"></i></a> <span id="tableRows">4</span> <a id="incRows" class="btn btn-sm btn-default"><i class="fa fa-plus"></i></a>';
```

Comment:
I think this was from a prior PR that hadn't run the build script. Looks good!

Reply:
Yeah, that's from the build script.
Could you also take a look at https://github.com/publiclab/PublicLab.Editor/pull/368
Thanks :)